#############
# ROS-Stuff #
#############

pixels_topic: '/pixels_relative'

pixels_marker_topic: '/fcnn_pixels_marker'


#################
# Visualization #
#################

pixels_namespace: 'fcnn_pixels'

